apps/cowswap-frontend/src/modules/notifications/hooks/useNotificationSettingsPopoverDismissal.ts

```diff
@@ -0,0 +1,42 @@
+import { useAtom } from 'jotai'
+import { atomWithStorage } from 'jotai/utils'
+import { useCallback, useMemo } from 'react'
+
+import { getJotaiMergerStorage } from '@cowprotocol/core'
+
+interface NotificationSettingsPopoverDismissalState {
+  dismissedAt: number | null
+}
+
+const STORAGE_KEY = 'notification-settings-popover-dismissed'
```

Comment:
Same comment as previous PR

```suggestion
const STORAGE_KEY = 'notificationSettingsPopoverDismissed:v0'
```